client/components/App.jsx

```diff
@@ -1,33 +1,124 @@
 import React from 'react';
-import TopComponent from './TopComponent.jsx'
+import TopComponent from './topComponent.jsx';
+import BottomComponent from './bottomComponent.jsx';
+import PositiveReviews from './positiveReviews.jsx';
+import NegativeReviews from './negativeReviews.jsx';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-    	reviews: []
+    	reviews: [],
```

Comment:
You've already mentioned this as a comment, but just to follow up, the spacing with your codes should be addressed